nextjsUIapp/package.json

```diff
@@ -9,25 +9,24 @@
     "lint": "next lint"
   },
   "dependencies": {
+    "aws-sdk": "^2.1692.0",
+    "cross-spawn": "^7.0.5",
```

Comment:
Assuming this is a sub-dependency of something? Or it is unused and can be removed?

Reply:
`aws-sdk` is being used in the data-release page. The data release page fetches the data from AWS s3 and make it available in the brainkb website.

Reply:
`cross-spawn` I believe is sub-dependency, I didn't add it explicitly.